### PR TITLE
fix: improve handling of pvc path flag

### PR DESCRIPTION
**Pull request by peternhale**
### What does this PR do?
Improves handling of the path flag for package version create
### What issues does this PR fix or reference?
@W-11903179@ GH issue

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,18 +6,19 @@
...
-    "@salesforce/packaging": "0.0.46",
+    "@oclif/core": "^1.18.0",
+    "@salesforce/command": "^5.2.12",
+    "@salesforce/core": "^3.31.10",
+    "@salesforce/kit": "^1.7.0",
+    "@salesforce/packaging": "0.1.1",
+    "chalk": "^4.1.2",
     "tslib": "^2",
-    "chalk": "^4.1.2"
+    "typescript-optional": "^2.0.1"
```
Comment (shetzel) [inline package.json:16]: I see this as a dependency burden for syntactic sugar.

File: src/commands/force/package/beta/version/create.ts [hunk 2]
```diff
@@ -221,4 +209,24 @@ export class PackageVersionCreateCommand extends SfdxCommand {
     }
     return result;
   }
+
+  private resolvePackageIdFromFlags(): string {
```
Comment (shetzel) [inline src/commands/force/package/beta/version/create.ts:213]: `PackageVersion.create()` handles all of this already right? That's what I see when looking at library code for `packageVersionCreate.packageVersionCreate()`.

**Review by shetzel — CHANGES_REQUESTED** [review]
Looks to me like we can remove a bunch of code in package beta version create command and rely on the library.

File: package.json (hunk 1 above)
Comment (peternhale) [reply package.json:16]: I will disagree.

Comment (peternhale) [reply package.json]: Will remove

**Review by shetzel — APPROVED** [review]

**Review by WillieRuemmele — APPROVED** [review]

**Pull request merged**